puppet/manifests/default.pp

```diff
@@ -12,6 +32,35 @@
   include seed_stack::load_balancer
 
   include docker_registry
+
+  include glusterfs_common
+
+  gluster_peer { 'standalone.seed-stack.local':
+    require => Package['glusterfs-server'],
```

Comment:
I'd prefer to not have it here as it seems a little confusing-- it being present here but not elsewhere.